lib/Zonemaster/LDNS/Packet.pm

```diff
@@ -127,6 +135,19 @@ Get or set the EDNS version in the packet. For incoming packets, returns 0 if
 the packet does not have an OPT pseudo-RR and 0 if it's an EDNS0 packet. It's
 thus rather pointless until such time as EDNS1 is defined.
 
+=item set_edns_present
+
+Set edns_present flag to true.
+
+This flag can be set when creating a packet with EDNS corner cases data that
+could not be detected by need_edns/has_edns methods.
+
+When set, need_edns/has_edns methods return true value.
```

Comment:
This is no longer true, right?

Reply:
We are talking about need_edns/has_edns methods.